pkg/scheduler/algorithm/predicates/predicates.go

```diff
@@ -149,8 +149,17 @@ var (
 		CheckServiceAffinityPred, MaxEBSVolumeCountPred, MaxGCEPDVolumeCountPred, MaxCSIVolumeCountPred,
 		MaxAzureDiskVolumeCountPred, MaxCinderVolumeCountPred, CheckVolumeBindingPred, NoVolumeZoneConflictPred,
 		CheckNodeMemoryPressurePred, CheckNodePIDPressurePred, CheckNodeDiskPressurePred, MatchInterPodAffinityPred}
+	checkAllPredicates = false
```

Comment:
Use global var does not sound like a good approach to me which could introduce unnecessary complexity, could you explain a bit more about the benefit?

Reply:
Is any other way to short-circuit predicate functions in `GeneralPredicates` if not using global var?  I think restructuring `type FitPredicate func` is not a good choice.